Rendera/src/main.cpp

```diff
@@ -62,42 +71,84 @@ int main(){
     Camera cam(cam_origin, port_width, port_height, img_width, img_height, focal_length);
 
     //ppm file
-    std::ofstream image("sphere.ppm");
```

Comment:
There is no need to comment out these lines because we are using Git, which stores the file histories in such a way that no work is lost. Remove these lines which you commented.